tests/functional/adapter/test_python_model.py

```diff
@@ -3,16 +3,20 @@
 from dbt.tests.util import run_dbt, write_file
 import dbt.tests.adapter.python_model.test_python_model as dbt_tests
 
-TEST_SKIP_MESSAGE = "Skipping the Tests since Dataproc serverless is not stable. " \
-                    "TODO: Fix later"
+
+@pytest.fixture(scope="module")
```

Comment:
Super bizarre that this also fails on serverless, have we tried the others on cluster mode?

Reply:
They all work on cluster mode. @ChenyuLInx tested this as well. 2 out of 3 working isn't giving confidence here.

Reply:
Yes locally on cluster all tests passed. The failure is likely due to Dataproc not dbt.

Reply:
Re-running to make sure this isn't a transient error.